components/clickToCopy.js

```diff
@@ -0,0 +1,64 @@
+import copy from 'copy-to-clipboard';
+import Button from './button';
+
+export default class ClickToCopy extends React.Component {
+  constructor() {
+    super();
+    this.state = {
+      copied: false
+    };
+  }
+  copyNow(t) {
+    copy(t);
+    this.setState({
+      copied: true
+    });
+  }
+  componentWillReceiveProps() {
```

Comment:
componentWillReceiveProps will make copied back to false is when user change input in the root component.

If user change content, then the system show the user the updated content has not yet been copied.